frontend/wallet/schema.graphql

```diff
@@ -1,3 +1,68 @@
+type Interval {
+  start: String! # timespan
+  end: String! # timespan
+}
+
+type Histogram {
+  values: [Int!]!
+  intervals: [Interval!]!
+  underflow: Int!
+  overflow: Int!
+}
+
+type RpcPair = {
+  dispatch: Histogram
+  impl: Histogram
+}
+
+type RpcTimings = {
+  getStagedLedgerAux: RpcPair!
+  answerSyncLedgerQuery: RpcPair!
+  getAncestry: RpcPair!
+  transitionCatchup: RpcPair!
+}
+
+type Histograms {
+  rpcTimings: RpcTimings!
+  externalTransitionLatency: Histogram
+  acceptedTransitionLocalLatency: Histogram
+  acceptedTransitionRemoteLatency: Histogram
+  snarkWorkerTransitionTime: Histogram
+  snarkWorkerMergeTime: Histogram
+}
+
+type ConsensusConfiguration {
+  delta: Int!
+  k: Int!
+  c: Int!
+  cTimesK: Int!
+  slotsPerEpoch: Int!
+  slotDuration: Int!
+  epochDuration: Int!
+  acceptableNetworkDelay: Int!
+}
+
+type DaemonStatus {
+  numAccounts: Int
+  blockCount: Int
+  uptimeSecs: Int!
+  ledgerMerkleRoot: String
+  stagedLedgerHash: String
+  stateHash: String
+  commitId: String! # git sha
+  confDir: String!
+  peers: [String!]!
+  userCommandsSent: Int!
+  runSnarkWorker: Boolean!
+  syncStatus: SyncStatus!
+  proposePubkeys: [String!]! @examples(values: [["PUB_KEY_E9873DF4453213303DA61F2", "PUB_KEY_F1173DF4453213303DA61F2"]])
```

Comment:
should the examples be base58 these days?

Reply:
we can follow on with a separate PR that changes all the key examples in the file later